app/views/cost_of_living_landing_page/show.html.erb

```diff
@@ -28,54 +28,42 @@
   end
 %>
 
-<header class="cost-of-living-hub__header">
-  <%= render "shared/browse_header", { margin_bottom: 7, padding_top: 0 } do %>
-    <%= render 'govuk_publishing_components/components/breadcrumbs', {
-      breadcrumbs: breadcrumbs,
-      collapse_on_mobile: true,
-      inverse: true
-    } %>
+<%= render "shared/browse_breadcrumbs", {
+  breadcrumbs: breadcrumbs,
+  collapse_on_mobile: true,
+  inverse: true
+} %>
 
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds">
-        <%= render "govuk_publishing_components/components/title", {
-          title: content[:header][:heading],
-          inverse: true,
-          margin_top: 4,
-          margin_bottom: 6,
-        } %>
-      </div>
-    </div>
+<%= render "shared/browse_header", { margin_bottom: 7, padding_top: 0 } do %>
 
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-two-thirds">
-        <%= render "govuk_publishing_components/components/lead_paragraph", {
-          text: content[:header][:lede],
-          inverse: true,
-          margin_bottom: 2,
-        } %>
-      </div>
-    </div>
+  <%= render "govuk_publishing_components/components/title", {
+    title: content[:header][:heading],
+    inverse: true,
+    margin_top: 6,
```

Comment:
This adds quite a lot of margin on top of the component on mobile. 

<img width="372" alt="Screenshot 2022-08-31 at 18 37 35" src="https://user-images.githubusercontent.com/5007934/187743728-636e047e-5cae-4dd2-b5a7-43e8cf08dfd5.png">

The browse header actually uses the heading component here, not the title component. I'm wondering if switching to it here would fix this.

Reply:
Good spot, thanks. This structure for the title and lead paragraph has now been changed to match with browse/topics pages. This fixes the too-big top margin on mobile and improves overall consistency.